OpenRA.Mods.Cnc/Traits/Spreads.cs

```diff
@@ -0,0 +1,74 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Cnc.Traits
+{
+	[Desc("A trait that controls forest fires. Any actor with this trait enabled has a chance to affect others with this trait.")]
```

Comment:
Code is generic but description says it is for forest fires.
```suggestion
	[Desc("An actor with this trait enabled has a chance to affect others with this trait. Example usage is forest fires.")]
```